src/main/java/uk/gov/hmcts/reform/idam/api/fr/idm/manage/user/model/IdmUser.java

```diff
@@ -661,6 +661,12 @@ public void setConsentedMappings(List<List<Object>> consentedMappings) {
     this.consentedMappings = consentedMappings;
   }
 
+  @JsonProperty("_meta")
+  private void fetchMetadataValues(Metadata metadata) {
+    Optional.ofNullable(metadata).ifPresent(m -> {
+      this.lastChanged = m.getLastChanged();
+    });
+  }
```

Comment:
to keep this pojo simple, should we add meta as attribute and have getter/setter for "fake" lastChanged?
just in case we have more attributes in meta. also wondering if idm will lose some values the way it is now.
This is what I was thinking:
```
@JsonProperty("_meta")
private Metadata meta;
private LastChanged lastChanged;

getMeta(...){...}
setMeta(...){...}
getLastChanged(...) { meta?meta.getLastChanged : null }
setLastChanged(...) { meta=meta?meta.setLastChanged : new Meta().setLastChanged }
```

Not sure if we already have a pattern for this style, what do you think @nikola-naydenov-hmcts ?